File: BasicAndPrinting/Assignment5/ALittleQuiz/ALittleQuiz.java
//BasicAndPrinting ALittleQuiz 40pts Nico Liao
import java.util.Scanner;
public class ALittleQuiz {
    public static void main(String[] args) {
        Scanner keyboard = new Scanner(System.in);
		int ans;
        int correct = 0;
		System.out.println("Q1) What is the capital of Alaska?");
		System.out.println("	1) Melbourne");
		System.out.println("	2) Anchorage");
		System.out.println("	3) Juneau");
		ans = keyboard.nextInt();
        if (ans == 3) {
            System.out.println("Correct, next");
            correct++;
        }else {
            System.out.println("Incorrect, it was 3");
        }
        System.out.println("Q2) Can you store the value \"cat\" in a variable of type int?");
        System.out.println("	1) yes");
        System.out.println("	2) no");
        ans = keyboard.nextInt();
        if (ans == 2) {
            System.out.println("Correct, next");
            correct++;
        }else {
            System.out.println("Incorrect, it was 1");
        }
        System.out.println("Q3) What is the result of 9+6/3?");
        System.out.println("	1) 5");
        System.out.println("	2) 11");
        System.out.println("	3) 15/3");
        ans = keyboard.nextInt();
        if (ans == 2) {
            System.out.println("Correct, next");
            correct++;
        }else {
            System.out.println("Incorrect, it was 2");
        }
        System.out.println("You got " + correct + " out of 3 correct");
    }
}

File: C++/Conversion2/ALittleQuiz/ALittleQuiz.cpp
//IfStatement ALittleQuiz 40pts Nico Liao
#include <iostream>
using namespace std;
int main()
{
    int ans;
    int correct;
    cout << "What is the capital of Alaska" << endl;
    cout << "	1) Melbourne" << endl;
    cout << "	2) Anchorage" << endl;
    cout << "	3) Juneau" << endl;
    cin >> ans;
    if (ans == 3) {
        cout << "Correct, next" << endl;
        correct++;
    }
    else{
        cout << "Incorrect, it was 3" << endl;
    }
    cout << "Q2) Can you store the value \"cat\" in a variable of type int?" << endl;
    cout << "	1) Yes" << endl;
    cout << "	2) No" << endl;
    cin >> ans;
    if (ans == 2) {
        cout << "Correct, next" << endl;
        correct++;
    }
    else{
        cout << "Incorrect, it was 2" << endl;
    }
    cout << "Q3) What is the result of 9+6/3?" << endl;
    cout << "	1) 5" << endl;
    cout << "	2) 11" << endl;
    cout << "	3) 15/3" << endl;
    cin >> ans;
    if (ans == 2) {
        cout << "Correct, next" << endl;
        correct++;
    }
    else{
        cout << "Incorrect, it was 2" << endl;
    }
    cout << "You got " << correct << " out of 3" << endl;
    return 0;
}